spec/e2e/scope_spec.rb

```diff
@@ -97,6 +97,26 @@
     end
   end
 
+  describe 'Person.scope :great_students, -> (identifier, score) { where("#{identifier}.score > ?", score)' do
+    before(:each) do
+      Person.scope :great_students, -> (identifier, score) { where("#{identifier}.score > ?", score || 41) }
```

Comment:
This lambda syntax is not valid on jruby 1.7 https://github.com/jruby/jruby/issues/2704

It should just work by removing the empty space between the `->` and `(identifier, scope) ...`:

```ruby
Person.scope :great_students, ->(identifier, score) { where("#{identifier}.score > ?", score || 41) }
```